modules/binding-infra-node-workloads-using-taints-tolerations.adoc

```diff
@@ -1,19 +1,25 @@
 // Module included in the following assemblies:
 //
 // * machine_management/creating-infrastructure-machinesets.adoc
+// * post_installation_configuration/cluster-tasks.adoc
 
 [id="binding-infra-node-workloads-using-taints-tolerations_{context}"]
 = Binding infrastructure node workloads using taints and tolerations
 
 If you have an infra node that has the `infra` and `worker` roles assigned, you must configure the node so that user workloads are not assigned to it.
 
+[IMPORTANT]
+====
+It is recommended to preserve the dual `infra,worker` label that is created for infra nodes and use taints and tolerations to manage nodes user workloads are scheduled on. If you remove the `worker` label from the node, you must create a custom pool to manage it; a node with a label other than `master` or `worker` is not recognized by the MCO without a custom pool. Maintaining the `worker` label allows the node to be managed by the default worker machine config pool, and the `infra` label communicates to the cluster that it does not count toward the total number of subscriptions.
+====
+
 .Prerequisites
 
 * Configure additional `MachineSet` objects in your {product-title} cluster.
 
 .Procedure
 
-. Use the following command to add a taint to the infra node to prevent scheduling user workloads on it:
+. If you did not configure a taint on your machine set, use the following command to add a taint to the infra node to prevent scheduling user workloads on it:
```

Comment:
I don't think this should be an 'if' statement; however I understand why we may have it written this way. 

IMO; this recommendation is 'requiring' that 'infra' nodes be 'tainted' - to ensure that workloads are not scheduled to these nodes.

Reply:
@sferich888 As you likely know, the _Creating infrastructure machine sets_ assembly, one of which where this topic appears, has several places suggesting to add the `node-role.kubernetes.io/infra:NoSchedule` taint. That, I assume is why it's an `if` statement, as the user might have already done this. I can change this to _If you have not configured a taint..._ 

PTAL at the change in my new PR: https://github.com/openshift/openshift-docs/pull/33057/commits/9069b2a88353049c6de646a10887068e72f00e9d